tests/unit/setAnniversaryDate.unitTest.ts

```diff
@@ -0,0 +1,262 @@
+import { TestResultsService } from "../../src/services/TestResultsService";
+import { expect } from "chai";
+
+describe("TestResultsService calling setTestAnniversaryDate", () => {
+    let testResultsService: TestResultsService | any;
+    let MockTestResultsDAO: jest.Mock;
+
+    beforeEach(() => {
+        MockTestResultsDAO = jest.fn().mockImplementation(() => {
+            return {};
+        });
+        testResultsService = new TestResultsService(new MockTestResultsDAO());
+    });
+
+    afterEach(() => {
+        MockTestResultsDAO.mockReset();
+    });
+
+    context("if testResult is trl", () => {
+        const hgvTestResultWithExpiryDate = JSON.parse(" {\n" +
```

Comment:
these look basically the same, with  only one or two fields changed - perhaps consider extracting to a file, and mutating copies?